pytest.ini

```diff
@@ -57,7 +57,7 @@ filterwarnings =
   error
 
   # FIXME: Address the deprecation warning in Python 3.10 and revert this ignore
-  ignore:There is no current event loop:DeprecationWarning:proxy.core.acceptor.threadless
+  # ignore:There is no current event loop:DeprecationWarning:proxy.core.acceptor.threadless
```

Comment:
Why didn't you just delete these lines?

Reply:
Totally, I initially didn't delete coz I wasn't confident if this would work.  Later i missed removing them altogether before merge.